draft-ietf-sframe-enc.md

```diff
@@ -467,25 +467,25 @@ o [Optional] A signature algorithm
 
 This document defines the following ciphersuites:
 
-
 | Value  | Name                           | Nh | Nk | Nn | Reference |
-|:-------|:-------------------------------|:---|:---|:---|:----------|
-| 0x0001 | AES\_CM\_128\_HMAC\_SHA256\_8  | 32 | 16 | 12 | RFC XXXX  |
-| 0x0002 | AES\_CM\_128\_HMAC\_SHA256\_4  | 32 | 16 | 12 | RFC XXXX  |
-| 0x0003 | AES\_GCM\_128\_SHA256          | 32 | 16 | 12 | RFC XXXX  |
-| 0x0004 | AES\_GCM\_256\_SHA512          | 64 | 32 | 12 | RFC XXXX  |
+|:-------|:-------------------------------|:---|----|:---|:----------|
+| 0x0001 | AES\_CM\_128\_HMAC\_SHA256\_10 | 32 | 16 | 12 | RFC XXXX  |
```

Comment:
isn't it a bit inconsistent to use bits in SHA256 and bytes in 8? wouldn't it be better to rename it to

AES\_CM\_128\_HMAC\_SHA256\_80

Reply:
OK, done